src/Model/lbsProcessor.py

```diff
@@ -160,8 +186,20 @@ def get_load(self):
         """Return computed total load on processor
         """
 
+        return sum([o.get_time() for o in self.objects.union([self.sentinel])])
```

Comment:
If the intention here is to return sum of time of all object including sentinel it should look like that:

```
proc_obj_with_sentinel = self.objects.add(self.sentinel)
return sum([o.get_time() for o in proc_obj_with_sentinel])
```

Reply:
@marcinwrobel1986 it should really be a set **union** so as to not replicate elements in the sum

Reply:
So basically something like that:
```
self.objects.add(self.sentinel)
return sum([o.get_time() for o in self.objects])
```

Reply:
Then maybe
```
return sum([o.get_time() for o in self.objects]) + self.sentinel.get_time()
```

Reply:
Sure by why is it better than what I did with the `union()`?  Is it more efficient in terms of alg. complexity?

Reply:
I guess it's just readability, subjective thing, don't worry about that. You can keep it as you want Philippe :) I will understand that.

Reply:
I agree that the `union()` syntax is not very intuitive. It lends to believing that it is actually performing the union on the set that is given to it (while it is not). Not too sure which one reads easier, between both formulations